### PR TITLE
Option to include inlinks in commonscrawl dump

### PR DESCRIPTION
**Pull request by thammegowda**
This PR enhances the CommonCrawlDumper with an optional CLI argument to accept linkdb path.
When this option is supplied, an additional field called "inlinks" will be included in the dump files.

We enhanced this as part of MEMEX crawler for analysing crawl link graph from dump files we believe this may be useful for other nutch users as well.

CC @sujen1412 @chrismattmann

### CODE REVIEW

File: src/java/org/apache/nutch/tools/AbstractCommonCrawlFormat.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import java.net.URLEncoder;
 import java.net.UnknownHostException;
```
Comment (lewismc) [inline src/java/org/apache/nutch/tools/AbstractCommonCrawlFormat.java]: Hi @thammegowda can you pleaae reformat this entire file to have 2 space indents here? I can review this PR once that is done. Thanks

Comment (thammegowda) [inline src/java/org/apache/nutch/tools/AbstractCommonCrawlFormat.java]: @lewismc 
As you had noticed it was previously formatted with tabs. I reformatted to 2 spaces.
I used IntelliJ Idea to reformat, I hope that followed all the style rules in `eclipse-codeformat.xml`. 
Please review and also let me know if you find any issues.

**Comment by thammegowda** [comment]
UPDATE: The entire ` src/java/org/apache/nutch/tools/AbstractCommonCrawlFormat.java` file has been reformatted to 2 space indents instead of tabs so you will see lot of Additions and Deletions.

**Comment by lewismc** [comment]
Dynamite

On Saturday, April 30, 2016, Thamme Gowda <notifications@github.com> wrote:

> UPDATE: The entire
> src/java/org/apache/nutch/tools/AbstractCommonCrawlFormat.java file has
> been reformatted to 2 space indents instead of tabs so you will see lot of
> Additions and Deletions.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/nutch/pull/106#issuecomment-216004708>
>


-- 
*Lewis*

**Comment by chrismattmann** [comment]
I went ahead and merged this thanks @thammegowda and @lewismc !

**Pull request merged**